examples/documents/example.proto

```diff
@@ -23,6 +23,9 @@ message ExampleDocument {
   bytes value_ignored = 8 [
     (proofs.exclude_from_tree) = true
   ];
+  bytes value_not_hashed = 9 [
+    (proofs.hashed_field) = true
```

Comment:
`value_not_hashed` field is actually hashed? based on the extension flag

Reply:
The naming is confusing, not_hashed = it should not be hashed.